src/dom/element.ts

```diff
@@ -803,13 +803,19 @@ export class Element extends Node {
 
   toggleAttribute(rawName: string, force?: boolean) {
     const name = String(rawName?.toLowerCase());
-    if (force === undefined) {
-      force = !this.hasAttribute(name);
-    }
-    if (force) {
-      this.setAttribute(name, "");
-    } else {
-      this.removeAttribute(name);
+    switch (this.hasAttribute(name)) {
```

Comment:
nit: can you change this from `switch (this.hasAttribute(name))` block to an `if (this.hasAttribute(name))`